configure.py

```diff
@@ -1289,10 +1289,12 @@ def configure_zstd(build_dir, mode):
                  ' -lstdc++fs', ' -lcrypt', ' -lcryptopp', ' -lpthread',
                  maybe_static(args.staticboost, '-lboost_date_time -lboost_regex -licuuc'), ])
 
-xxhash_dir = 'xxHash'
+pkgconfig_libs = [
+    'libxxhash',
+]
 
-if not os.path.exists(xxhash_dir) or not os.listdir(xxhash_dir):
-    raise Exception(xxhash_dir + ' is empty. Run "git submodule update --init".')
+args.user_cflags += ' ' + ' '.join([pkg_config(lib, '--cflags') for lib in pkgconfig_libs])
```

Comment:
What is this change?

Reply:
xxhash_dir no longer exists so we shouldn't check that it exists.
On the other hand, we should use pkgconfig to add the correct cflags and ldflags.

Reply:
Oh, nice. I didn't notice you had this new pkgconfig_libs technique. That makes me wonder, why is xxhash special in that regard? Why don't we use pkgconfig also for zlib and a dozen other libraries? Is it because we actually do the same thing differently in cmake instead of here in configure.py? Shouldn't we do these things the same? Are you starting a move back from cmake to configure.py? :-(

Reply:
It's not special. It comes with pkgconfig support so I used that. We should use it for the other libraries too, but that's not part ofthis patch.

Reply:
"We should use it for the other libraries too" - I agree, but my point was that we actively took the opposite direction. For example, instead of just using "pkg-config --libs liblz4" in configure.py as we did in the past, we have cmake/Findlz4.cmake which,  in 70 lines of unintelligable mumbo-jumbo basically uses pkg-config too...
Anyway, this patch is fine. I'll commit.